### ISSUE TITLE
cx logs should follow by default

### ISSUE THREAD

**Issue by nzoschke** [convox/praxis#245]
It's a preference but I think `cx logs` should tail app logs by default.

**Comment by sc-chad** [comment]
I agree with this.  Find my self doing `cx logs` .. crap `cx logs --follow` way to often.

**Comment by mwarkentin** [comment]
It'd be nice to have a simple `--no-follow` in order to disable - I always forget the syntax to not follow on `convox`.
